components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/configuration/HumanTaskServerConfiguration.java

```diff
@@ -71,6 +71,9 @@
 
     private boolean uiRenderingEnabled = false;
 
```

Comment:
Lets rename these two variables to emailNotificationEnabled and smsNotificationEnabled